src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/UpdateResourcesValidation.feature

```diff
@@ -30,9 +28,8 @@ Feature: Resources "Update" Operation validations
                         "shortDescription": "Sick Leave"
                     }
                   """
-             Then it should respond with 204
+             Then it should respond with A04
```

Comment:
This should break.

Reply:
You modified the feature file, but did not commit the modified `.feature.cs` file. The feature file is not read at runtime - only at compile time, when generating the `.feature.cs`.

Reply:
I just ran a build locally with this file and the `.feature.cs` _did not change_, which surprises me. Looking into the reason.